mod_tariffs.asciidoc

```diff
@@ -918,8 +959,8 @@ A Price Component describes how a certain amount of a certain dimension being co
 |Property |Type |Card. |Description
 
 |type |<<mod_tariffs_tariffdimensiontype_enum,TariffDimensionType>> |1 |The dimension that is being priced
-|price |<<types.asciidoc#types_number_type,number>> |1 |Price per unit (excl. VAT) for this dimension.
-|vat |<<types.asciidoc#types_number_type,number>> |? |Applicable VAT percentage for this tariff dimension. If omitted, no VAT is applicable. Not providing a VAT is different from 0% VAT, which would be a value of 0.0 here.
+|price |<<types.asciidoc#types_number_type,number>> |1 |Price per unit for this dimension. This is including or excluding taxes according to the `tax_included` field of the Tariff that this PriceComponent is contained in.
+|vat |<<types.asciidoc#types_number_type,number>> |? |Applicable VAT percentage for this tariff dimension. If omitted, no VAT is applicable. 
```

Comment:
Shouldn't this support different types of taxes as well? See also type Price. If not, it this the total percentage of taxes or just the VAT?